test/utilities/apexUtils.ts

```diff
@@ -17,23 +17,28 @@ export async function createApexClass(name: string, classText: string, breakpoin
 
   // Set the name of the new Apex Class
   await inputBox.setText(name);
+  await pause(Duration.seconds(1));
```

Comment:
@daphne-sfdc All the pauses? What are we missing that can be waited for?

Reply:
I noticed that without the pauses, there is a flapper in setup that causes the Apex class to not be created.  See Mac and Ubuntu in this run: https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/11768545442